src/util/message.h

```diff
@@ -275,55 +275,64 @@ class messaget
     return m;
   }
 
-  // in lieu of std::endl
-  static mstreamt &endl(mstreamt &m)
+  class commandt
   {
-    static_cast<std::ostream &>(m) << std::endl;
-    return m;
+  public:
+    commandt(unsigned _command) : command(_command)
```

Comment:
Missing `explicit` - unless that omission was intentional, in which case I'd wonder even more whether a `typedef` might just be good enough.